docs/AboutUs.adoc

```diff
@@ -3,53 +3,43 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
-{empty} +
-We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+Inifity Book was developed by the https://github.com/CS2103JAN2018-W11-B3[CS2103JAN2018-W11-B3] team. +
+We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore], from the CS2103 Tutorial W11.
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Sashankh Chengavalli Kumar
+image::Sashankh.jpg[width="150", align="left"]
+{empty}[https://github.com/ChengSashankh[github]] [<<johndoe#, portfolio>>]
 
-Role: Project Advisor
+Role: Developer +
+Responsibilities: Documentation
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Nguyen Thi Viet Ha
+image::Viet Ha.jpg[width="150", align="left"]
+{empty}[https://github.com/deeheenguyen[github]] [<<johndoe#, portfolio>>]
 
 Role: Team Lead +
 Responsibilities: UI
 
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Huynh Thanh Duc Anh
+image::Anh.jpg[width="150", align="left"]
+{empty}[https://github.com/anh2111[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Data
+Responsibilities: Code Quality
 
 '''
 
-=== Johnny Roe
+=== Nguyen Dinh Dung
 image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+{empty}[https://github.com/tiny767[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Dev Ops + Threading
-
-'''
-
-=== Benson Meier
-image::yl_coder.jpg[width="150", align="left"]
-{empty}[http://github.com/yl-coder[github]] [<<johndoe#, portfolio>>]
-
-Role: Developer +
-Responsibilities: UI
+Responsibilities: Deliverables and Deadlines
 
```

Comment:
Okay, but if you stick strictly to the given requirements, it states that "The photo of a team member should be doc/images/githbub_username_in_lower_case.png e.g. docs/images/damithc.png."